lib/craneMultiBuild.nix

```diff
@@ -1,19 +1,25 @@
-{ mkFenixToolchain
-, craneLib
-, mapWithToolchains
-, mkTarget
-, mkStdToolchains
-, lib
-}:
-let craneLib' = craneLib; in
-{ toolchains ? mkStdToolchains { }
-, profiles ? [ "dev" "ci" "release" ]
-, craneLib ? craneLib'
-}: outputsFn:
-let
-  profilesFn = craneLib: craneLib.mapWithProfiles outputsFn profiles;
+{
+  mkFenixToolchain,
```

Comment:
What are you using for formatting? I'm still locked with `nixpkgs-fmt`, and planning to switch to `nixfmt` after it stops being unstable (`nixfmt-rfc-style`)

Reply:
I'm using [alejandra](https://github.com/kamadorueda/alejandra), do you want me to re-format the whole thing with `nixpkgs-fmt` instead?

Reply:
Yes, since everything else is formatted using nixpkgs-fmt and I think there are even git pre-commit checks for it.